### ISSUE TITLE
GenomeDiagram example Proux_et_al_2002_Figure_6.py needs updating

### ISSUE THREAD

**Issue by TheChymera** [biopython/biopython#563]
[This example file](https://github.com/biopython/biopython/blob/82272b1f36e3b7dd28296bde518b248db946b590/Doc/examples/Proux_et_al_2002_Figure_6.py) fails with:

```
Traceback (most recent call last):
  File "test.py", line 127, in <module>
    assert record.name not in feature_sets
AssertionError
```

**Comment by peterjc** [comment]
Are you running the example unmodified? How did you get the sample GenBank files?

**Comment by TheChymera** [comment]
Yes, unmodified. Got them from the GenBank website.

**Comment by peterjc** [comment]
I can't yet explain the error message you have (unless it is a simple mistake like saving a record with the wrong filename?).

Note for ``NC_003212`` via the website you must ask for the FULL record in GenBank format, otherwise you get a summary record with no genes and a ``CONTIG`` entry instead of the sequence.

The annotation of this bacteria has been revised and the gene names in this script are now recorded under ``old_locus_tag``. However, even with the locus tag detection tweaked https://github.com/biopython/biopython/commit/8457989f090df0e304e83cabff6d8cdc1a61a88e , there is something different about the number of features (and thus the colours pre-assigned to them). I suspect something on the annotation has changed subtly...

**Comment by sbliven** [comment]
This should be fixed as of 94c9d7e right?

**Comment by peterjc** [comment]
Yep, thank you!
